src/vm/hints/secp/secp_utils.rs

```diff
@@ -48,6 +61,21 @@ pub fn pack(d0: &BigInt, d1: &BigInt, d2: &BigInt, prime: &BigInt) -> BigInt {
         .sum()
 }
 
+pub fn pack_from_var_name(
+    name: &str,
+    ids: &HashMap<String, BigInt>,
+    vm: &mut VirtualMachine,
```

Comment:
Do we need this to be mutable?

Reply:
resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/333/commits/f9552216b25097ece469f70cff35ee8620aa5b64